packages/react-scripts/config/paths.js

```diff
@@ -52,7 +88,10 @@ module.exports = {
   testsSetup: resolveApp('src/setupTests.js'),
   appNodeModules: resolveApp('node_modules'),
   ownNodeModules: resolveApp('node_modules'),
-  nodePaths: nodePaths
+  nodePaths: nodePaths,
+  publicUrl: getPublicUrl(resolveApp('package.json')),
```

Comment:
It's fine to give these more verbose names if that would make it easier to understand what they represent. Right now it's not super obvious to me what the difference is.